--- freeciv-1.14.1/configure.ac~	Mon Jul 26 09:34:38 2004
+++ freeciv-1.14.1/configure.ac	Mon Jul 26 09:34:38 2004
@@ -143,7 +143,7 @@
 dnl  AC_PROG_MAKE_SET
 
 dnl I18n support
-ALL_LINGUAS="da de en_GB es et fi fr hu it ja nl no pl pt pt_BR ro ru sv uk"
+ALL_LINGUAS="da de en_GB es et fi fr hu it ja nb nl pl pt pt_BR ro ru sv uk"
 AM_GNU_GETTEXT(,need-ngettext)
 
 dnl AM_GNU_GETTEXT doesn't have all features we want
--- freeciv-1.14.1/configure.in~	Mon Jul 26 09:34:46 2004
+++ freeciv-1.14.1/configure.in	Mon Jul 26 09:34:46 2004
@@ -142,7 +142,7 @@
 dnl  AC_PROG_MAKE_SET
 
 dnl I18n support
-ALL_LINGUAS="da de en_GB es et fi fr hu it ja nl no pl pt pt_BR ro ru sv uk"
+ALL_LINGUAS="da de en_GB es et fi fr hu it ja nb nl pl pt pt_BR ro ru sv uk"
 AM_GNU_GETTEXT(,need-ngettext)
 
 dnl AM_GNU_GETTEXT doesn't have all features we want
--- freeciv-1.14.1/configure~	Mon Jul 26 09:34:57 2004
+++ freeciv-1.14.1/configure	Mon Jul 26 09:34:57 2004
@@ -3468,7 +3468,7 @@
 
 
 
-ALL_LINGUAS="da de en_GB es et fi fr hu it ja nl no pl pt pt_BR ro ru sv uk"
+ALL_LINGUAS="da de en_GB es et fi fr hu it ja nb nl pl pt pt_BR ro ru sv uk"
 
 
         echo "$as_me:$LINENO: checking for strerror in -lcposix" >&5
