--- freeciv-2.1.1/bootstrap/freeciv.desktop.in~	2007-11-30 21:03:59.000000000 +0100
+++ freeciv-2.1.1/bootstrap/freeciv.desktop.in	2007-11-30 21:03:21.000000000 +0100
@@ -4,5 +4,6 @@
 Name[ca]=Freeciv
 Name[es]=Freeciv
 Name[fr]=Freeciv
-Name[no]=Freeciv
+Name[nb]=Freeciv
+Name[pl]=Freeciv
 Name[pt]=Freeciv
@@ -12,6 +13,7 @@
 Comment[da]=Strategispil inspireret af den menneskelige civilisations historie
 Comment[de]=Rundenbasiertes Strategiespiel, inspiriert durch die Geschichte der menschlichen Zivilisation
 Comment[fi]=Ihmiskunnan historian inspiroima vuoropohjainen strategiapeli
-Comment[no]=Strategispill inspirert av historien til menneskelig sivilisasjon
+Comment[nb]=Strategispill inspirert av historien til menneskelig sivilisasjon
+Comment[pl]=Turowa gra strategiczna zainspirowana historią ludzkiej cywilizacji
 Comment[pt]=Jogo de estratégia por turnos inspirado na História da civilização humana
 Comment[sv]=Turordningsbaserat strategispel inspirerat av den mänskliga
@@ -22,3 +22,4 @@
 Type=Application
 Categories=GTK;Game;StrategyGame;
 Keywords=strategy;simulation;civilization;tiles;history;mankind;multiplayer;
+# vi: encoding=utf-8
--- freeciv-2.2.0-beta1/bootstrap/freeciv-server.desktop.in~	2009-12-13 10:57:16.000000000 +0000
+++ freeciv-2.2.0-beta1/bootstrap/freeciv-server.desktop.in	2009-12-24 10:33:18.558453813 +0000
@@ -3,14 +3,16 @@
 Name[ca]=Freeciv server
 Name[es]=Freeciv server
 Name[fr]=Freeciv server
-Name[no]=Freeciv server
+Name[nb]=Freeciv server
+Name[pl]=Serwer Freeciv
 Name[pt]=Freeciv server
 Comment=Turn-based strategy game inspired by the history of human civilization
 Comment[ca]=Joc d'estratègia inspirat en la història de la civilització humana
 Comment[da]=Strategispil inspireret af den menneskelige civilisations historie
 Comment[de]=Rundenbasiertes Strategiespiel, inspiriert durch die Geschichte der menschlichen Zivilisation
 Comment[fi]=Ihmiskunnan historian inspiroima vuoropohjainen strategiapeli
-Comment[no]=Strategispill inspirert av historien til menneskelig sivilisasjon
+Comment[nb]=Strategispill inspirert av historien til menneskelig sivilisasjon
+Comment[pl]=Turowa gra strategiczna zainspirowana historią ludzkiej cywilizacji
 Comment[pt]=Jogo de estratégia por turnos inspirado na História da civilização humana
 Comment[sv]=Turordningsbaserat strategispel inspirerat av den mänskliga
 Exec=civserver
@@ -18,5 +20,6 @@
 StartupNotify=true
 Terminal=true
 Type=Application
-Categories=ConsoleOnly;Game;StrategyGame;
+Categories=GTK;Game;StrategyGame;
 Keywords=strategy;simulation;civilization;tiles;history;mankind;multiplayer;
+# vi: encoding=utf-8
